doc/map.md

```diff
@@ -35,15 +47,17 @@ U res = op(x1,x2,...,xN)
 
 ### Unary map
 
-An unary **Map** takes a data set and transforms each element in the data set by applying an unary function and generating a new data set.
+An unary **map** takes a data set and transforms each element in the data set by
+applying an unary function and generating a new data set.
 
-The only interface currently offered for this pattern is based in iterators (following the C++ standard library conventions):
+The only interface currently offered for this pattern is based in iterators
+(following the C++ standard library conventions):
 
   * The input data set is specified by two iterators.
   * The output data set is specified by an iterator to the start of the output sequence.
 
 ---
-**Example**
+**Example**: Squaring values in a vector.
```

Comment:
The example doubles the values in the vector instead of square them. 

The description or the example should be modified .